canvas_exercise/shapes_game/index.js

```diff
@@ -1,15 +1,88 @@
 window.addEventListener("load", function() {
+  //flow of what needs to happen:
```

Comment:
Good start and your code looks clean so far.  Try to add the seconds timer and stop the game after 30 seconds have passed.